src/olympia/shelves/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+
+from olympia.amo.models import ModelBase
+
+CHOICE_TYPES = (
+    ('category', 'category'),
+    ('collection', 'collection'),
+    ('extension', 'extension'),
+    ('recommended', 'recommended'),
+    ('search', 'search'),
+    ('theme', 'theme'),
+)
+
+
+class Shelf(ModelBase):
+    title = models.CharField(max_length=200, unique=True)
+    shelfType = models.CharField(
```

Comment:
😄 too much javascript - python prefers snake_case rather than camelCase for it's class attributes.

Reply:
the same for the other properties

Reply:
😅 - updated!